jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1483,7 +1462,7 @@ void checkSecurity(Class<?> definingClass, Class<?> referenceClass, int modifier
 					VMLangAccess vma = getVMLangAccess();
 
 					ClassLoader referenceClassLoader = referenceClass.getClassLoader();
-					if (isWeakenedLookup() || !doesClassLoaderDescendFrom(referenceClassLoader, accessClass.getClassLoader())) {
+					if (isWeakenedLookup() || !com.ibm.oti.util.Util.doesClassLoaderDescendFrom(referenceClassLoader, accessClass.getClassLoader())) {
```

Comment:
you can - `import static com.ibm.oti.util.Util.doesClassLoaderDescendFrom;`. then, you can directly call `doesClassLoaderDescendFrom`.